experiments/tf_trainer/keras_cnn/run_ml_engine.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Random note: seeing such a similar file is motivating me to figure out some way to make this common